CHANGELOG.md

```diff
@@ -1,3 +1,87 @@
+# 78.0.0
+
+This major release promotes the Shape libraries from private to ready, as well as promoting Action Sheet to ready by migrating it from the MaterialComponentsBeta podspec to the MaterialComponents main podspec.
+
```

Comment:
Highlighting the fact that we're increase our snapshotting test coverage is a good win for clients as well; I suggest we add something like:

> We've increased our snapshot test coverage across multiple components.

Reply:
Done.